pangeo/values.yaml

```
@@ -54,6 +54,8 @@ jupyterhub:
 #      guarantee: 2G
 
 dask-gateway:
+  enabled: true
```

Could you add a comment here describing what this does? Or is it obvious enough to not need one?

```suggestion
  # set enabled to False to not install dask-gateway.
  enabled: true
```